README.md

```diff
@@ -61,17 +69,19 @@ Team-Mate is an independent company dedicated to helping students and trainees m
 
 ## Important Notes
 
-- **Starter Project**: This project provides a basic structure and examples to help you get started. The provided components and hooks are just examples. You are encouraged to modify and extend them according to your needs.
-- **Flexibility**: You are not restricted to the current architecture or the packages used. Feel free to make any modifications to suit your approach to solving the challenge.
-
-## Overview
-
-The provided components and hooks demonstrate how to set up a WebSocket connection and manage chat interactions. The `WebSocketContext` provides context for managing the WebSocket connection and chat state, while `useConsumer` and `usePublisher` hooks handle message consumption and publishing.
-
-## Contributions
-
-We welcome contributions and suggestions! Feel free to fork the repository, make improvements, and submit a pull request.
+**Features in the works**
+- Delete session
+- Remove session leakage
+- User accounts
+- Landing page
+- Audio handling
+- update knowlage base
```

Comment:
Suggested Edit:
Enhance the knowledge base with more comprehensive content and up-to-date information.